lara-typescript/src/interactive-api-client/client.ts

```diff
@@ -14,6 +14,10 @@ interface IListenerMap {
   [key: string]: IRequestCallback[];
 }
 
+const parseJSONIfString = (data: any) => {
+  return typeof data === "string" ?  JSON.parse(data) : data;
```

Comment:
Do we need to worry about potential parse errors here?

Reply:
Not sure. I think it's pretty unlikely to happen. But if it does, we can't initialize interactive anyway. So, the only difference is whether JSON parse error is thrown or not. And whether we display some message to user. I think this error should NOT be handled gracefully, as this is not something that is expected. So, Rollbar should report it. We could add a message for users so it's easier to report issues manually too. But I would need to be an (annoying) alert I guess, as we can't do much more from iframe level without having access to interactive itself.